docs/book/src/user/quick-start.md

```diff
@@ -161,17 +161,17 @@ If you are unsure you can determine your computers architecture by running `unam
 
 Download for AMD64:
 ```bash
-curl -L {{#releaselink gomodule:"sigs.k8s.io/cluster-api" asset:"clusterctl-linux-amd64" version:"1.5.x"}} -o clusterctl
+curl -L {{#releaselink repo:"https://github.com/kubernetes-sigs/cluster-api" gomodule:"sigs.k8s.io/cluster-api" asset:"clusterctl-linux-amd64" version:"1.5.x"}} -o clusterctl
```

Comment:
I think this change is fine - but it's a required change for all downstream users of this tool.

Should I add a note to the provider migration guide?

Reply:
Yes let's add a note. 

Absolutely agree, nice and pragmatic way to get rid of the dependency and preserve the current feature